dynawo/sources/Launcher/CMakeLists.txt

```diff
@@ -41,7 +41,11 @@ TARGET_LINK_LIBRARIES(dynawo
                       ${LibArchive_LIBRARY}
                       ${ZLIB_LIBRARY}
                       "-rdynamic"
-                      )
+                    )
+
+if("${CMAKE_CXX_COMPILER_ID}" STREQUAL "GNU")
+  target_link_libraries(dynawo "-lrt")
```

Comment:
Please keep the coding style (capital letters)

Reply:
Why is this dependency needed?

Reply:
On old distrib with static compilation it is required
https://stackoverflow.com/questions/2418157/c-error-undefined-reference-to-clock-gettime-and-clock-settime

Reply:
The coding style in cmake is not well done everywhere and I want to change it to lower case as it is the new norm, capital letter is the old one and a rather aggresive one

Reply:
OK but currently it is the coding style we use, so we should keep it homogeneous

Reply:
Please do it everywhere